contrib/validatepr/validatepr.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -x
+
+#
+# This script is intended to help developers contribute to the podman project. It
+# checks various pre-CI checks like building, linting, man-pages, etc.  It is meant
+# to be run in a specific container environment.
+#
+
+# build all require incantations of podman
+echo "--"
+echo "$REMOTETAGS"
+echo "--"
+
+echo "Building windows ..."
+GOOS=windows CGO_ENABLED=0 go build -tags "$REMOTETAGS" -o bin/test.windows ./cmd/podman
+echo "Building darwin..."
+GOOS=darwin CGO_ENABLED=0 go build -tags "$REMOTETAGS" -o bin/test.darwin ./cmd/podman
```

Comment:
aren't there make targets for this?

Reply:
no

Reply:
That was my question too, and AFAICT there's a windows zip target triggered by `GOOS`, and a really complicated `OSX` task in Cirrus, but no simple Makefile targets. (I was surprised).

Reply:
Once upon a time I believe they existed but we kept building on the original targets so they got more complex, and now we don't have a simple one.